### PR TITLE
Remove comment blocks from module - Backlog Item 83064

### PR DESCRIPTION
**Pull request by JamJarchitect**
# Overview/Summary

Remove multi-line comments from the top of each module that contains module info as duplicated info (version, author etc.)

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. Removed comment blocks from ALZ Bicep modules
2. Updated Contributing guide to remove the comment block from Bicep authoring best practices

### Breaking Changes

1. Comment block removal from modules
2. Contributing guide updated to reflect removal of comment blocks

## Testing Evidence

image

image

## As part of this Pull Request I have

- [X] Read the [Contribution Guide](https://github.com/Azure/ALZ-Bicep/wiki/Contributing) and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/ALZ-Bicep/pulls)
- [N/A] Associated it with relevant [GitHub Issues](https://github.com/Azure/ALZ-Bicep/issues)
- [X] *(ALZ Bicep Core Team Only)* Associated it with relevant [ADO Items](https://aka.ms/alz/bicep/backlog)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/ALZ-Bicep/tree/main)
- [X] Performed testing and provided evidence.
- [N/A] Updated tests *(if required)* [Unit](https://github.com/Azure/ALZ-Bicep/blob/main/.github/workflows/bicep-build-to-validate.yml) - [Linting](https://github.com/Azure/ALZ-Bicep/tree/main/.github/workflows) - [E2E (End-To-End)](https://github.com/Azure/ALZ-Bicep/blob/main/tests/pipelines/bicep-build-to-validate.yml)
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Module READMEs, Wiki Docs etc.)
- [N/A] If relevant, created or updated Code Tours [here](https://github.com/Azure/ALZ-Bicep/blob/main/.vscode/tours)

### CODE REVIEW

File: docs/wiki/Contributing.md [hunk 1]
```diff
@@ -93,14 +93,6 @@ Throughout the development of Bicep code you should follow the [Bicep Best Pract
 - Each bicep file must contain the below multi-line comment at the very top of the file, with its details filled out:
```
Comment (jfaurskov) [inline docs/wiki/Contributing.md:93]: Could we re-phrase as this doesn't really make sense after removing the comment? and change ```bicep to where it is relevant?

**Review by jfaurskov — CHANGES_REQUESTED** [review]
Other than the comments on the contributing guide, it's looking good to me.

File: docs/wiki/Contributing.md (hunk 1 above)
Comment (JamJarchitect) [reply docs/wiki/Contributing.md]: Removed multi-line comment recommendation and adjusted the ```bicep section to targetScope = line. Commit [75bd299](https://github.com/Azure/ALZ-Bicep/pull/218/commits/75bd29944b21aa903768e194389112beb127a83c)

**Review by jfaurskov — CHANGES_REQUESTED** [review]

File: docs/wiki/Contributing.md [hunk 2]
```diff
@@ -90,18 +90,8 @@ Throughout the development of Bicep code you should follow the [Bicep Best Pract
   - Default values should also be documented in the appropriate location
 - Tab indents should be set to `2` for all Bicep files
 - Double line-breaks should exist between each element type section
-- Each bicep file must contain the below multi-line comment at the very top of the file, with its details filled out:
 
 ```bicep
```
[the remark this answers is not in the document]
Comment (JamJarchitect) [reply docs/wiki/Contributing.md]: Worked through comments with Jan on commit [dc1417f](https://github.com/Azure/ALZ-Bicep/pull/218/commits/dc1417f555b0162acd4fa321a0bd5ed4ac4351f1)

**Review by jfaurskov — COMMENTED** [review]
LGTM

**Review by jfaurskov — APPROVED** [review]

**Pull request merged**